### PR TITLE
Add a recipe for spyder-notebook

### PR DESCRIPTION
**Pull request by dalthviz**
This is a Spyder plugin for Jupyter notebook integration.

@ccordoba12 , @goanpeca , could you guys accept being maintainers? thanks :)

### CODE REVIEW

**Comment by goanpeca** [comment]
Yep :-)

**Comment by ccordoba12** [comment]
+1 from me too.

File: recipes/spyder-notebook/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  run:
+    - python
+    - spyder >=3.2
+    - notebook >=4.3
+    - qtpy
+    - requests
+    - psutil
+    - nbformat
+
```
Comment (goanpeca) [inline recipes/spyder-notebook/meta.yaml]: @ccordoba12 should we add some `>=` pinning here?

**Comment by goanpeca** [comment]
@dalthviz is this ready?

**Comment by dalthviz** [comment]
@goanpeca I think so :+1

**Comment by goanpeca** [comment]
@jakirkham merging  :-p ?

**Comment by jakirkham** [comment]
Thanks @dalthviz.

**Pull request merged**